packages/browserify/src/index.js

```diff
@@ -177,6 +177,10 @@ function getConfigurationFromPluginOpts(pluginOpts) {
       pluginOpts.scuttleGlobalThisExceptions
   }
 
+  if (!pluginOpts.resolve) {
+    pluginOpts.resolve = require('browser-resolve')
```

Comment:
not sure why moving the require was necessary, but it shouldn't change anything.

Reply:
Just a minor optimization so it's not pulled in if unused.